python/tests/test_metadata.py

```diff
@@ -1417,37 +1552,46 @@ def test_mutation(self):
             ),
         ]:
             self.assertEqual(
-                metadata.MetadataSchema(schema).decode_row(example), expected
+                metadata.MetadataSchema(schema).decode_row(example)["SLiM"], expected
             )
 
     def test_population(self):
         schema = {
             "codec": "struct",
             "type": "object",
             "properties": {
-                "subpopulationID": {"type": "integer", "binaryFormat": "i", "index": 0},
-                "femaleCloneFraction": {
-                    "type": "number",
-                    "binaryFormat": "d",
-                    "index": 1,
-                },
-                "maleCloneFraction": {
-                    "type": "number",
-                    "binaryFormat": "d",
-                    "index": 2,
-                },
-                "sexRatio": {"type": "number", "binaryFormat": "d", "index": 3},
-                "boundsX0": {"type": "number", "binaryFormat": "d", "index": 4},
-                "boundsX1": {"type": "number", "binaryFormat": "d", "index": 5},
-                "boundsY0": {"type": "number", "binaryFormat": "d", "index": 6},
-                "boundsY1": {"type": "number", "binaryFormat": "d", "index": 7},
-                "boundsZ0": {"type": "number", "binaryFormat": "d", "index": 8},
-                "boundsZ1": {"type": "number", "binaryFormat": "d", "index": 9},
-                "migrationRecCount": {
-                    "type": "integer",
-                    "binaryFormat": "d",
-                    "index": 10,
-                },
+                "SLiM": {
```

Comment:
Do we actually need this here? This is already an object without forcing the top level key, isn't it?

Reply:
The only table that needs this is `mutations` as otherwise its top-level type is `array`. I thought it looked better to be consistent across tables. I think we need to think about how this works with multiple tools adding to metadata - otherwise we'll end up with name clashes and people renaming existing entries to things like `old_X`.

Reply:
I don't think we should recommend every program having one key "programname" - that would be really tedious and error prone.

We don't need to get into this here anyway, as these are just test cases. Unless it's actually needed to make the test cases pass, lets delete the top-level key. For the mutations table the key should be something more informative than "SLiM" - this doesn't tell the read much about what's actually in there. We can make recommentations on how metadata should be structured when multiple tools are expected to update it later.

Reply:
Yep, I don't think a top-level "programme" key is right either in the general case. I've removed the top-level except for mutations where it is now `stacked_mutation_array`.